### ISSUE TITLE
Got error : "sign-in to confirm you are not a robot"

### ISSUE THREAD

**Issue by jerohnimaux** [libre-tube/LibreTube#6116]
### Steps to reproduce

* Log in a Piped instance (tried kavin.rocks, lunar.icu)
* Click on any video
* The message "Got error : "sign-in to confirm you are not a robot"" appears and the video doesn't play

### Expected behavior

The video should play

### Actual behavior

An error is displayed : Got error : "sign-in to confirm you are not a robot"

### LibreTube version

0.23.2

### Android version

GrapheneOS Android 14 build 2024060500

### Other details

Same issue happened for many different projects that get YouTube videos.
The issue was fixed by implementing support for OAuth2 tokens for YouTube : https://github.com/imputnet/cobalt/issues/551

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by kiss2u** [comment]
same issue here, guessing it is a typical youtube problem rather than a tech bug.

**Comment by TheSystemGuy1337** [comment]
YouTube has been going on a crusade against downloader utilities recently. See [this comment I made on this issue involving yt-dlp for additional details. In short, DON'T PASS COOKIES.](https://github.com/yt-dlp/yt-dlp/issues/10128#issuecomment-2155037169)

**Comment by gloriouspuffy** [comment]
Same issue but for me it only happens on projectsegfau.lt

**Comment by Sarvesh-N** [comment]
Same issue for me, This error occurs to me whenever I to to watch the video or try to download the video...
Duplicate of #

**Comment by Sarvesh-N** [comment]
Same issue for me, This error occurs to me whenever I try to watch the video or try to download the video...

**Comment by em3250n** [comment]
> Same issue for me, This error occurs to me whenever I try to watch the video

Same issue here

**Comment by P00RY** [comment]
So is there a fix or...?

**Comment by ImMALWARE** [comment]
Same for me, when i try to watch any video

**Comment by P00RY** [comment]
> So is there a fix or...?

NVM I just had to change the instance, if that doesn't work for someone I dunno what will.

**Comment by ImMALWARE** [comment]
> > So is there a fix or...?
> 
> NVM I just had to change the instance, if that doesn't work for someone I dunno what will.

what instance is working for you?

**Comment by AverySalsedo21** [comment]
Newpipe doesnt have this issue maybe the piped team should ask them what they did? I use libre tube mostly but newpipe is just my backup youtube client and it has basically no problem with playing videos . just trying to help out . i love libretube and prefer it

**Comment by Tintin232** [comment]
So, is there any solution yet? Which instance is most likely to work?

**Comment by Bnyro** [comment]
Tldr: Change the instance in the settings.

**Comment by Bnyro** [comment]
After https://github.com/libre-tube/LibreTube/pull/6381, you can go to
`Settings` -> `Instance` and enable `Disable Piped proxy` as well as `Local streams extraction` if your obligated to use a public instance.

Private instances are likely still working in most cases anyways.

I will publish a new release in some time.
